src/SprykerEco/Yves/Computop/Dependency/Service/ComputopToUtilEncodingServiceBridge.php

```diff
@@ -29,7 +29,7 @@ public function __construct($utilEncodingService)
      *
      * @return string|null
      */
-    public function encodeJson($value, $options = null, $depth = null)
+    public function encodeJson($value, $options = null, $depth = null): ?string
```

Comment:
* Roll back changes.
* The bridge method must have an equal method signature as a source.